shared/src/helpers/string/change-string-case/change-string-case.helper.ts

```diff
@@ -0,0 +1,20 @@
+import { paramCase, Options } from 'change-case';
+import { StringCase } from '~/common/enums/case/case.enum';
+
+const caseTypeToFn: Record<StringCase, typeof paramCase> = {
+  [StringCase.KEBAB_CASE]: paramCase,
+};
```

Comment:
```suggestion
const caseTypeToFn: Record<StringCase, typeof paramCase> = {
  [StringCase.KEBAB_CASE]: paramCase,
  [StringCase.SNAKE_CASE] snakeCase
};
```
we decided to use the snake-case for keys